.github/dependabot.yml

```diff
@@ -1,5 +1,14 @@
 version: 2
 updates:
+  - package-ecosystem: "gradle"
```

Comment:
might want to resync here, these changes don't seem right

Reply:
Yup, thanks I just fixed it